test/jasmine/tests/plot_interact_test.js

```diff
@@ -142,6 +150,119 @@ describe('Test plot structure', function() {
             });
         });
 
+        describe('scatter drag', function() {
+
+            var mock = require('@mocks/scatter_simple.json'),
```

Comment:
They have a large shared part but aren't identical in visuals or layout range: 10.json is a real boxplot while gl2d_10.json is just the non-boxplot part. But I can easily redo the numbers with 10.json, shouldn't take long.